.github/workflows/build_and_release.yml

```
@@ -0,0 +1,81 @@
+on:
+  workflow_dispatch: # Allows manual triggering
+    inputs:
+      tag:
+        description: 'Tag name'     
+        required: true
+      note:
+        description: 'Release note'
+        
+jobs:
+  build:
+    runs-on: ubuntu-latest  
```

yamllint

[error] trailing spaces

(trailing-spaces)